client/src/components/ViewModeToggle/ViewModeToggle.scss

```diff
@@ -0,0 +1,122 @@
+.viewmode-dropdown-toggle {
+  margin-left: auto;
+  font-size: 13px;
+
+  #splitModeButton {
+    &:disabled {
+      pointer-events: all;
+    }
+  }
+
+  .viewModeToggle-chosenView-title {
+    font-size: 10px;
+    line-height: 10px;
+    vertical-align: top;
+    white-space: normal;
+    width: 44px;
+    display: inline-block;
+    text-align: left;
+  }
+}
+
+.dropdown {
+
+  &.viewModeToggle {
+    position: absolute;
+    right: 10px;
+
+    .viewmode-dropdown-toggle {
+      &:before {
+        font-size: 18px;
+      }
+    }
+  }
+
+  .dropdown-menu.show {
+    min-width: 12rem;
+
+    .btn {
+      margin-left: 0;
+      border-radius: 0;
+
+      &:active {
+        &:before {
+          color: $white;
+        }
+      }
+    }
+
+    .viewMode-selected.btn {
+      &,
+      &:hover,
+      &:before,
+      &.highlighted,
+      &:focus {
+        background: $dropdown-link-active-bg;
+        color: $component-active-color;
+        border: 0;
+      }
+    }
+  }
+}
+
+.disabled-tooltip {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 37px;
+  background: rgba($white, 0);
+
+  .disabled-tooltip-span {
+    display: none;
+
+    &:before {
+      content: "";
+      border: 5px solid transparent;
+      border-top-color: $gray-700;
+      border-bottom-width: 0;
+      display: block;
+      width: 10px;
+      height: 5px;
+      position: absolute;
+      margin: auto;
+      top: 100%;
+      left: 0;
+      right: 0;
+    }
+  }
+
+  &:hover {
+    .disabled-tooltip-span {
+      display: block;
+      text-align: center;
+      background: $gray-700;
+      color: white;
+      padding: 4px 0;
+      position: absolute;
+      top: -24px;
+      left: -1px;
+      right: -1px;
+      border-radius: 3px 3px 0 0;
+    }
+  }
+}
+
+// When using the caret property, ReactStrap adds a .dropdown-toggle class to the DropdownToggle component, which in
+// turn changes the background colour. This behaviour is overriden here.
+.viewModeToggle {
+  &.dropdown {
+    &.show {
+      .btn-secondary.dropdown-toggle {
+        box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125), 0 0 0 0.2rem rgba(86, 107, 141, 0.5);
+        border-color: #e7ebf0;
+        background-color: transparent;
+        color: #566b8d;
```

Comment:
Is there any way we can swap these out for variables, or even `darken()` or `lighten()` calls on variables that are close to these values already?